spec/src/modules/browse.js

```diff
@@ -468,7 +489,7 @@ describe(`ConstructorIO - Browse${bundledDescriptionSuffix}`, () => {
   });
 
   describe('getBrowseResultsForItemIds', () => {
-    const ids = ['fc00a355-1e91-49a1-b6c6-c8c74c50259d', '1f32b500-f397-4faa-90e0-4c4625b3e3b5'];
```

Comment:
These old ids don't give any results because they don't exist anymore.

https://ac.cnstrc.com/browse/items?c=cio-mocha&key=ZqXaOfXuBWD4s3XzCI1q&i=69a02bae-335f-4297-a576-c71d5f679c01&s=1&fmt_options%5Bhidden_fields%5D=testField&_dt=1645227005919&ids=fc00a355-1e91-49a1-b6c6-c8c74c50259d&ids=1f32b500-f397-4faa-90e0-4c4625b3e3b5